api/src/controllers/events/index.ts

```diff
@@ -1 +1,3 @@
-export { clickEvent } from '@/src/controllers/events/click-event';
\ No newline at end of file
+export { clickEvent, paginatedClickEvents } from '@/src/controllers/events/click-event';
+export { inputEvent, paginatedInputEvents } from '@/src/controllers/events/input-event';
+export { visitEvent, paginatedVisitEvents } from '@/src/controllers/events/visit-event';
```

Comment:
should only need to export the first member of each of those files with a unified object for routing:

i.e., see https://github.com/GTBitsOfGood/bog-analytics/pull/21/files#r1493344769